tools/integration_tests.py

```diff
@@ -110,7 +110,7 @@ def generate(self, test_filename):
 
 def main():
     parser = argparse.ArgumentParser()
-    parser.add_argument("--filter", help="Run specific tests")
+    parser.add_argument("--filter", type=str, help="Run specific tests")
```

Comment:
I think we can use the same arg parser (parse_test_args) as everywhere else (use .pattern and add --executable).

Reply:
I will as soon as I figure out filtering :+1: